e2e/tests/home/home-page.spec.ts

```diff
@@ -32,6 +32,11 @@ describe('Home page ui tests', () => {
     await GoogleLoginPage.logIn(page);
 
     const cards = await WorkspaceCard.getAllCards(page);
+    if (cards.length === 0) {
```

Comment:
It feels very odd for a test to short-circuit depending on the structure of the page. If we believe that the assertions further down in the test case are important to effectively QA the app, then this early return should be considered a test breakage!

I can think of a couple options here:

1) As part of the set-up of this overall test file, include a call which creates at least one workspace.
2) Change the cleanup script to leave at least one workspace, so we never have to early-return from this test case.